_rules/html-page-lang-valid-bf051a.md

```diff
@@ -53,7 +53,7 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-The `lang` attribute specified is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) and has a valid primary language subtag.
+This `html` element has a `lang` attribute whose value is a [valid][valid language subtag] [primary language subtag][].
```

Comment:
Missed a `[]` after valid language subtag?

Reply:
No. The ref is `[valid][valid language subtag]` which is correct.
And the (new) test agrees with me 😄